API/models/schemas/layer_schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "OGrEE Tag Schema",
+  "type": "object",
+  "properties": {
+    "slug": {
+      "type": "string",
+      "$ref": "refs/types.json#/definitions/slug"
+    },
+    "filters": {
+      "type": "object"
+    },
+    "applicability": {
+      "type": "string"
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "slug",
+    "filters",
+    "applicability"
+  ],
+  "examples": [
+    {
+      "slug": "racks",
+      "filters": {"category": "rack"},
+      "applicability": "/Physical/BASIC/A/R1/*"
```

Comment:
From the point of view of the api, this does not exact, it is specific t the CLI (APP does not know it either)

Reply:
I do not understand this comment well, I understand that the API does not understand paths, but it is an information that I need to keep of the layer, since it is information that was completed by the user to know which is the applicability of the layer. Can you think of another way to do it?

Reply:
Small typo in the last comment there. I would recommend converting it to the API ID (BASIC.A.R1.*), a wildcard * it's something that the API already has, adding the ** should be possible too.

Reply:
fixed in 518f7560 and f2582970